### ISSUE TITLE
[startup crash] UninitializedPropertyAccessException: lateinit property quickActionSheetView has not been initialized

### ISSUE THREAD

**Issue by agi90** [mozilla-mobile/fenix#4521]
https://sentry.prod.mozaws.net/operations/fenix/issues/6107480/

```
UninitializedPropertyAccessException: lateinit property quickActionSheetView has not been initialized
    at org.mozilla.fenix.browser.BrowserFragment.access$getQuickActionSheetView$p(BrowserFragment.kt:1)
    at org.mozilla.fenix.browser.BrowserFragment$onViewCreated$9.invoke(BrowserFragment.kt:2)
...
(8 additional frame(s) were not displayed)

lateinit property quickActionSheetView has not been initialized
```

**Comment by agi90** [comment]
I've hit this twice in the last two days, it's a startup crash (probably when Fenix is cold starting). I'll update this if it keeps happening.

**Comment by agi90** [comment]
This is my stacktrace, since I don't see my report on Sentry:

```
FATAL EXCEPTION: main
Process: org.mozilla.fenix, PID: 8147
kotlin.UninitializedPropertyAccessException: lateinit property quickActionSheetView has not been initialized
    at io.reactivex.plugins.RxJavaPlugins.throwUninitializedPropertyAccessException(RxJavaPlugins.java:2)
    at org.mozilla.fenix.browser.BrowserFragment.access$getQuickActionSheetView$p(BrowserFragment.kt:1)
    at org.mozilla.fenix.browser.BrowserFragment$onViewCreated$9.invoke(BrowserFragment.kt:2)
    at mozilla.components.lib.state.ext.FragmentKt$consumeFrom$1.invokeSuspend(Fragment.kt:8)
    at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:2)
    at kotlinx.coroutines.DispatchedTask.run(Dispatched.kt:15)
    at android.os.Handler.handleCallback(Handler.java:873)
    at android.os.Handler.dispatchMessage(Handler.java:99)
    at android.os.Looper.loop(Looper.java:214)
    at android.app.ActivityThread.main(ActivityThread.java:7032)
    at java.lang.reflect.Method.invoke(Native Method)
    at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:494)
    at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:965)
```

**Comment by csadilek** [comment]
This was fixed as part of https://github.com/mozilla-mobile/fenix/issues/4412. So, should not happen anymore on latest builds (uplifted to 1.2 as well).

**Comment by csadilek** [comment]
Closing as fixed in 1.2-rc2 and latest Nightly. Please re-open if you still see it.

**Comment by agi90** [comment]
Just confirming that I haven't seen this at all this week :)
